src/store/modules/resources.js

```diff
@@ -18,6 +19,7 @@ export default {
   state: reactive({
     resources: [],
     resourceDetails: {},
+    resourceForm: {},
```

Comment:
I was wondering about that. If we had more components that would need to use this information, it would make sense to save it in the store but in this case we are using it only in UserResourceForm. We are already emitting the call for handle-submit to the parent view (Add, Remove, Transfer), so it won't hurt to pass arguments as well. And it would remove the storage object handling (clearing/fetching of data) from the views and the component.

Reply:
I was wondering about that. If we had more components that would need to use this information, it would make sense to save it in the store but in this case we are using it only in UserResourceForm. We are already emitting the call for handle-submit to the parent view (Add, Remove, Transfer), so it won't hurt to pass arguments as well. And it would remove the storage object handling (clearing/fetching of data) from the views and the component.

Reply:
fixed